arm-asazure/2016-05-06/swagger/asazure.json

```diff
@@ -0,0 +1,553 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": { "$ref": "#/definitions/AnalysisServicesResource" }
+          },
+          "201": {
+            "description": "InProgress",
+            "schema": { "$ref": "#/definitions/AnalysisServicesResource" }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "description": "Deletes the specified Analysis Services server.",
+        "tags": [ "Servers" ],
+        "operationId": "Servers_Delete",
+        "parameters": [
+          {
+            "name": "analysisServerName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Name of the Analysis Services server"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": { "$ref": "#/definitions/AnalysisServicesResource" }
+          },
+          "204": {
+            "description": "No Content"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "patch": {
+        "description": "Updates the current state of the specified Analysis Services server as specified by the request",
+        "tags": [ "Servers" ],
+        "operationId": "Servers_Update",
+        "parameters": [
+          {
+            "name": "analysisServerName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Name of the Analysis Services server"
+          },
+          {
+            "name": "updateRequest",
+            "in": "body",
+            "required": true,
+            "schema": { "$ref": "#/definitions/AnalysisServicesResourceParameters" },
+            "description": "Request object for updating the server"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": { "$ref": "#/definitions/AnalysisServicesResource" }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AnalysisServices/servers": {
+      "get": {
+        "description": "Gets all the Analysis Services servers for the given resource group",
+        "tags": [ "Servers" ],
+        "operationId": "Servers_ListByResourceGroup",
```

Comment:
Apply the pageable extension. In this case, the nextLink would be set to null in the extension. Thus, you still get the benefit of flattening the "value" property which is of type array. The generated code will return an `IEnumerable<AnalysisServicesResource>`.

`"x-ms-pageable":{ "nextLinkName": null }`